.github/workflows/circleci.yml

```diff
@@ -0,0 +1,23 @@
+# To enable this workflow on a fork, comment out:
+#
+# if: github.repository == 'napari/docs'
+
+name: CircleCI artifact redirector
+
+on: [status]
+jobs:
+  circleci_artifacts_redirector_job:
+    runs-on: ubuntu-latest
+    if: "github.event.context == 'ci/circleci: build-docs'"
+    permissions:
+      statuses: write
+    name: Run CircleCI artifacts redirector
+    # if: github.repository == 'napari/docs'
```

Comment:
Should this be un-commented? (based on the line at the top of the file `to enable this workflow on a fork comment out:`

Reply:
Yeah - so if this is commented the action also runs on forks (the case now that I'm running this on my fork). We should probably remove this before merging. If you folks are ready for that (i.e. no other reviews are necessary) I can do that.

Reply:
For me, the `code` seems correct and it works on your fork...
This doesn't directly affect the website or any end users, so I'd be game to merge sooner rather than later and then if need be tweak something? Dunno if that makes sense in the grand scheme of things—will defer to others with more experience, like you and @jni

Reply:
Why? By default, a contributor will not have circleCI configured, so the workflow will not be triggered. But could a contributor allow testing without creating PR? I would rather see a limited list of branches.

Reply:
Sometimes people work on their Fort until get a satisfying solution to not depend on the test queue of the napari organization.

Reply:
Hmm, fair enough. But I think docs contributors may be less savvy and more like end-users, rather than developers, so this PR is a nice quality of life improvement where they can see the rendered docs online.

Reply:
It is my understanding this will only run in their forks if they do sign up with CircleCI - otherwise they won't have the appropriate credentials.

Reply:
I don't have an opinion here. It's easy enough for a contributor to edit that line... Is this ready other than this discussion?